src/main/java/de/tum/cit/aet/artemis/iris/service/pyris/dto/chat/PyrisChatStatusUpdateDTO.java

```diff
@@ -6,6 +6,18 @@
 
 import de.tum.cit.aet.artemis.iris.service.pyris.dto.status.PyrisStageDTO;
 
+/**
+ * A DTO to send chat-based Pyris pipeline status updates to the client.
+ *
+ * @param result      Iris' textual response, if any.
+ * @param stages      The pipeline execution progress described in a list of stages
+ * @param suggestions A list of suggested next inputs for the user to click on
+ */
+// @formatter:off
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-public record PyrisChatStatusUpdateDTO(String result, List<PyrisStageDTO> stages, List<String> suggestions) {
-}
+public record PyrisChatStatusUpdateDTO(
+        String result, // TODO: Wrap in Optional for API clarity
+        List<PyrisStageDTO> stages, // TODO: Wrap in Optional for API clarity
```

Comment:
Does a list actually have to be wrapped in an optional? Shouldn't we always expect lists to be empty and then because of Jackson null?

Reply:
That's a fair point, I suppose it makes no difference in that case. I'll remove it.